plugins/georeplication/store-utils.go

```diff
@@ -118,7 +118,7 @@ func getSSHPublicKeys(volname string) ([]georepapi.GeorepSSHPublicKey, error) {
 	}
 
 	if resp.Count < 1 {
-		return nil, errors.New("SSH Public Keys not found")
+		return nil, gderrors.ErrPublicSSHKeyNotFound
```

Comment:
As this error is not used anywhere else, doesn't have to be an error constant.

Reply:
Just to mantain consistency in how we are handling errors, I thought of adding ErrPublicSSHKeyNotFound and ErrDaemonNotFound